setup.py

```diff
@@ -20,6 +20,7 @@ def get_version():
         "hist>=1.2",
     ],
     "test": [
+        "awkward>=1.0.0,<2",
```

Comment:
```suggestion
        "awkward>=1.9.0rc1",
```

Have to specify the rc version, otherwise pip will take the latest non-rc release. Awkward 1.8.0 doesn't have enough v2 development in it.

Also, the main branch can drop `<2` qualifiers.